frontend/src/App.vue

```diff
@@ -379,7 +379,7 @@ html {
   }
 }
 
-#comparison-details, .metabolite-table, .model-table, .reaction-table, .subsystem-table {
+#comparison-details, .metabolite-table, .model-table, .reaction-table, .subsystem-table, .gembrowser-table {
   .main-table tr td.td-key, #ed-table tr td.td-key {
     width: 150px;
```

Comment:
I think `.gembrowser-table` should not have a fixed width, but instead leave the width to be decided by the parent component.

Reply:
It looks like this was done mostly to replicate the old CSS selectors of `.metabolite-table` and `.subsystem-table`, which could now be removed I think.

Reply:
Yeah  `.metabolite-table` and `.subsystem-table` should go. I'm not fond of removing `width: 150px` now though, that's kind of a major layout change which messes up the current layout. But I'm up for creating a new issue for it if you would like that

Reply:
> I'm not fond of removing width: 150px now though

👍🏻 @inghylt let's leave that for some other refactoring session - I'm not tempted to record this in an issue.

However, following @e0's previous comment, it looks like all the CSS selectors on that line are used for the exact same purpose. I think it's better to have one and the same class for everything.

Reply:
@mihai-sysbio  Good point. I think though that since `comparison-details` is not applied to a table, but all of the rest are, I would like to have `#comparison-details, .table-template` or similar

Reply:
sounds good!